api/quotes.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Quote = require('../models/quote');
+const router = require('express').Router();
+const HTTPStatus = require('http-status');
+
+router.get('/quotes', listQuotes);
+router.post('/quotes', createQuote);
+router.get('/quotes/:id', getQuote);
+router.put('/quotes/:id', updateQuote);
+router.delete('/quotes/:id', deleteQuote);
+
+module.exports = router;
+
+function listQuotes(req, res, next) {
+  Quote.find()
+    .then(quote => res.status(HTTPStatus.OK).send(quote))
+    .catch(err => next(err));
+}
+
+function getQuote(req, res, next) {
+  Quote.findById(req.params.id)
+    .then(quote => res.status(HTTPStatus.OK).send(quote))
+    .catch(err => next(err));
+}
+
+function createQuote(req, res, next){
+  let { quote, author } = req.body;
+  Quote.create({ quote, author })
```

Comment:
What about security aspect of this, body can contain garbage, and Quote.create saves it to base directly??